cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -196,16 +204,18 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
     reconfigure_vm(small_vm, orig_config)
 
 
-@pytest.mark.manual
 @pytest.mark.tier(1)
 @pytest.mark.parametrize('change_type', ['sockets', 'memory'])
-def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
-    """
+@pytest.mark.provider([VMwareProvider], override=True)
```

Comment:
Does this test apply for RHEV too ?

Reply:
No, we already have test for RHEV 
https://github.com/ManageIQ/integration_tests/blob/8a8f3f954b9e76c32ae443dea724d1a34a1de7a3/cfme/tests/infrastructure/test_vm_reconfigure.py#L180

Reply:
Test steps seems same .. can we reuse RHEV test for Vmware too and mark it to run for both providers?

Reply:
@sshveta In vmware we enabling hot_plugin and in RHV we don't need that also in wmware we can't revert the changes just like we doing for RHV
https://github.com/ManageIQ/integration_tests/blob/8a8f3f954b9e76c32ae443dea724d1a34a1de7a3/cfme/tests/infrastructure/test_vm_reconfigure.py#L195

Reply:
@niyazRedhat, either way this sounds like a code duplication.  
you can add if provider.one_of and run or don't run provider specific code.